src/p2p/behaviour.rs

```diff
@@ -515,6 +524,31 @@ impl<Types: IpfsTypes> Behaviour<Types> {
             }
         }
     }
+
+    pub fn dht_get(&mut self, key: Key) -> SubscriptionFuture<KadResult, String> {
+        self.kad_subscriptions
+            .create_subscription(self.kademlia.get_record(&key, Quorum::One).into(), None)
```

Comment:
setting the quorum to one seems a bit strange choice but probably helps passing the tests.

Should `K` be used here instead, maybe a FIXME or TODO comment? The HTTP API docs for these methods does not at least mention being able to specify the quorum. At least looking at `ipfs dht get --help` it discusses finding multiple values instead of the first and selecting the "best" where "best" is defined only for ipns records.

Reply:
good point :+1:; I left this decision to the caller for now